neurom/features/morphology.py

```diff
@@ -164,8 +164,55 @@ def trunk_vectors(morph, neurite_type=NeuriteType.all):
             for n in iter_neurites(morph, filt=is_type(neurite_type))]
 
 
+def _str_to_plane(plane):
+    """Transform a plane string into a list of coordinates."""
+    if plane is not None:
+        coords = []
+        plane = plane.lower()
+        if "x" in plane:
+            coords.append(COLS.X)
+        if "y" in plane:
+            coords.append(COLS.Y)
+        if "z" in plane:
+            coords.append(COLS.Z)
+    else:  # pragma: no cover
+        coords = [COLS.X, COLS.Y, COLS.Z]
+    return coords
+
+
+def _sort_angle(p1, p2):
```

Comment:
Should it be rather?
```
"""Angle between p1 and p2 to sort vectors."""
def _between_angle(p1, p2):
```
name `sort_angle` makes user think that it is for sorting of angles.

Reply:
Yeah true